airflow-core/src/airflow/utils/dag_cycle_tester.py

```diff
@@ -24,8 +24,8 @@
     from airflow.sdk.definitions.dag import DAG
 
 warnings.warn(
-    "`airflow.utils.dag_cycle_tester` module is deprecated and "
-    "will be removed in Airflow 3.2. Please use `dag.check_cycle()` method instead.",
+    "`airflow.utils.dag_cycle_tester` module is deprecated and and will be removed in a future release."
+    "Please use `dag.check_cycle()` method instead.",
```

Comment:
I assume the intention was that it will be removed in 3.x rather than in 4.0?
if we change it we may forget to do it